R/get_ds_rt.R

```diff
@@ -1,19 +1,20 @@
 #' Perform direct standardization
 #'
 #' @description Compute a directly standardized rate given the counts of events,
-#' the size of each defined population and the size of each group of the standard
-#' population.
+#' the size of each defined population and the size of each group of the
+#' standard population.
 #'
 #' @param counts A numeric vector containing the frequency of events.
 #' @param popn A numeric vector containing the size of each defined population.
 #' @param std_popn A numeric vector containing the size of each group of the
 #' standard population.
 #' @param scale,power Provide either `scale`, a positive integer, to specify the
 #' scale, or `power`, a non-negative integer, to specify the power of 10 to use
-#' when expressing the result. If neither argument is supplied, `scale = 1` will be
-#' applied.
-#' @param output_type If `"rate"` (default), the directly standardized rate is returned.
-#' If `"counts"`, the expected number of events for each stratum is returned.
+#' when expressing the result. If neither argument is supplied, `scale = 1` will
+#' be applied.
+#' @param output_type If `"rate"` (default), the directly standardized rate is
```

Comment:
I hope that the two sentences suggest that "rate" and "counts" are valid arguments to use here. I'm hesitant to include additional details since this function's documentation is already lengthy.

Reply:
Agree with @reikookamoto above and in line with other documentation where the type can be inferred from the documented list of valid values (e.g. `"rate"` and `"counts"` here, with the quotes). Examples in the documentation also show this.